### DISCUSSION TITLE
Executing .\requirements.bat doesn't work?

### DISCUSSION THREAD

**Discussion by Arc1233** [winstxnhdw/lc-hax#247] [Q&A]
I downloaded Microsoft .Net, then Git. Then cloned the repository, which worked. I try entering ".\requirements.bat" into Git and it says command not found. 

What am I doing wrong here?

**Comment by winstxnhdw** [comment]
Restart your computer.

**Reply by Arc1233** [reply]
Restarted and still gives the same message.
![Screenshot (2)](https://github.com/winstxnhdw/lc-hax/assets/156495008/0ea8aeac-3575-40fa-a576-cb064caa9537)

**Reply by winstxnhdw** [reply]
Yeah, of course you can't. You are trying to run a CMD command in Git Bash. Just open up the folder and run it directly or use the CMD terminal.

**Reply by Arc1233** [reply]
Thank you!! It works now
